clang/test/dpct/cmake_migration/case_028/input.cmake

```diff
@@ -0,0 +1,10 @@
+# this doesn't provide pkg-config
+# suggest to assign BLAS_INCLUDE_DIRS on your own
+set(GGML_HEADERS_CUDA ggml-cuda.h)
```

Comment:
Can we use general name instead of name from another project?

Reply:
Okay, will update it, in the next updated PR.